apps/vs-code-designer/src/app/commands/createNewCodeProject/cloudToLocalHelper.ts

```diff
@@ -0,0 +1,54 @@
+import type { ConnectionReferenceModel } from '@microsoft/vscode-extension-logic-apps';
+
+export function extractConnectionDetails(connections: ConnectionReferenceModel): any {
+  const SUBSCRIPTION_INDEX = 2;
+  const MANAGED_API_LOCATION_INDEX = 6;
+  const MANAGED_CONNECTION_RESOURCE_GROUP_INDEX = 4;
+
+  const details = [];
+  const managedApiConnections = connections['managedApiConnections'];
+  if (managedApiConnections) {
+    for (const connKey in managedApiConnections) {
+      if (Object.prototype.hasOwnProperty.call(managedApiConnections, connKey)) {
+        const api = managedApiConnections[connKey]['api'];
+        const connection = managedApiConnections[connKey]['connection'];
+        if (api?.id && connection?.id) {
+          const idPath = api['id'];
+          const connectionIdPath = connection['id'];
+          const apiIdParts = idPath.split('/');
+          const connectionIdParts = connectionIdPath.split('/');
+          if (apiIdParts) {
+            const detail = {
+              WORKFLOWS_SUBSCRIPTION_ID: apiIdParts[SUBSCRIPTION_INDEX],
+              WORKFLOWS_LOCATION_NAME: apiIdParts[MANAGED_API_LOCATION_INDEX],
+              WORKFLOWS_RESOURCE_GROUP_NAME: connectionIdParts[MANAGED_CONNECTION_RESOURCE_GROUP_INDEX],
+              WORKFLOWS_TENANT_ID: '72f988bf-86f1-41af-91ab-2d7cd011db47',
```

Comment:
Is this a constant that we need to keep? is this something that is going to change depending on the user?

Reply:
We do not need to keep this, I put it in because I thought it was needed to get the Access Token.